diff -urN glut-3.7/linux/Makefile glut-3.7.new/linux/Makefile
--- glut-3.7/linux/Makefile	2006-04-16 22:23:07.000000000 +0200
+++ glut-3.7.new/linux/Makefile	2006-04-16 22:38:54.000000000 +0200
@@ -87,8 +87,8 @@
       IXXINCLUDES = -i '<X11/Fresco/enter-scope.h>'
 
           INCROOT = /usr/X11R6/include
-        USRLIBDIR = /usr/X11R6/lib
-         SHLIBDIR = /usr/X11R6/lib
+        USRLIBDIR = /usr/X11R6/lib64
+         SHLIBDIR = /usr/X11R6/lib64
        LINTLIBDIR = $(USRLIBDIR)/lint
           MANPATH = /usr/X11R6/man
     MANSOURCEPATH = $(MANPATH)/man
@@ -150,7 +150,7 @@
      STD_INCLUDES =
   STD_CPP_DEFINES = -traditional -Dlinux -D__i386__ -D_POSIX_SOURCE -D_BSD_SOURCE -D_GNU_SOURCE -DX_LOCALE
       STD_DEFINES = -Dlinux -D__i386__ -D_POSIX_SOURCE -D_BSD_SOURCE -D_GNU_SOURCE -DX_LOCALE
- EXTRA_LOAD_FLAGS = -Wl,-rpath,/usr/X11R6/lib
+ EXTRA_LOAD_FLAGS = -Wl,-rpath,/usr/X11R6/lib64
   EXTRA_LDOPTIONS =
   EXTRA_LIBRARIES =
              TAGS = ctags
diff -urN glut-3.7/progs/Glut.cf glut-3.7.new/progs/Glut.cf
--- glut-3.7/progs/Glut.cf	2006-04-16 22:23:07.000000000 +0200
+++ glut-3.7.new/progs/Glut.cf	2006-04-16 22:41:52.000000000 +0200
@@ -104,14 +104,14 @@
 #endif
 
 #ifndef AlphaArchitecture
-DEPGLUT = /usr/lib/libglut.so
+DEPGLUT = /usr/lib64/libglut.so
 GLUT = -lglut
 #endif
-DEPMUI = /usr/lib/libmui.a
+DEPMUI = /usr/lib64/libmui.a
 MUI = -lmui
-DEPGLE = /usr/lib/libgle.a
+DEPGLE = /usr/lib64/libgle.a
 GLE = -lgle
-DEPGLSMAP = /usr/lib/libglsmap.a
+DEPGLSMAP = /usr/lib64/libglsmap.a
 GLSMAP = -lglsmap
 
 CXXEXTRA_INCLUDES = -I/usr/include/CC $(EXTRA_INCLUDES)
